### PR TITLE
create db via dappctrl db-create command

### PR DESCRIPTION
**Pull request by ubozov**
Resolve BV-1027.

### CODE REVIEW

File: data/command.go [hunk 1]
```diff
@@ -27,6 +30,12 @@ func ExecuteCommand(args []string) error {
 	}
 
 	switch args[0] {
+	case "db-create":
+		f := readFlags(args)
+		if err := createDatabase(f.connection); err != nil {
+			panic(fmt.Sprintf("failed to create database %s", err))
```
Comment (ababo) [inline data/command.go]: I would write `panic("failed to create database: " + err.Error())`.

File: data/command.go [hunk 2]
```diff
@@ -94,3 +103,50 @@ func initData(connStr string) error {
 		return nil
 	})
 }
+
+func createDatabase(connStr string) error {
+	db, err := sql.Open("postgres", connStr)
```
Comment (ababo) [inline data/command.go]: Please reuse routines in the `data`-package.

File: data/command.go (hunk 1 above)
Comment (ubozov) [reply data/command.go]: updated

File: data/command.go (hunk 2 above)
Comment (ubozov) [reply data/command.go]: updated

**Pull request merged**